### ISSUE TITLE
The component behaves incorrectly when an after image changes / loads with a delay

### ISSUE THREAD

**Issue by villyaraujo** [sneas/img-comparison-slider#5]
Hello,

I did the installation for Angular according to the documentation and it is with error to pass the image using the mouse. What should I do to resolve?

**Comment by sneas** [comment]
Hi Villy,

Thank you for your contribution. When I was trying to reproduce the problem, I discovered the library had not been updated for the latest Angular. I released a new version which is 2.3.0.

I've created an example app for you: https://github.com/sneas/example-img-comparison-slider-angular. I hope it will help.

Let me know if you still have any problems with the library. Also, an error message text would be helpful.

Feel free to reopen this issue if the problem is still there. Or create a new issue if you've discovered more problems with the library.

**Comment by villyaraujo** [comment]
Hello, 

I'm using ionic v3, just load the before image, the after image only loads if you inspect it in the browser. Besides, I don't have control of the divider with the mouse, when I touch it it disappears.

What could be happening?

Thanks for the support

**Comment by sneas** [comment]
What version of Angular are you using?

**Comment by villyaraujo** [comment]
Hello,

My Angular version:

Angular CLI: 9.1.4
Node: 10.15.0
OS: darwin x64
Angular: 5.2.11
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic
Ivy Workspace: Yes

Thanks for the support

**Comment by sneas** [comment]
That's an interesting bug. I'll think of a solution. Meanwhile you can use a workaround:

```html
<img-comparison-slider>
  <img slot="before" [src]="before" (load)="onImageLoaded()">
  <img slot="after" [src]="after" (load)="onImageLoaded()">
</img-comparison-slider>
```

```ts
onImageLoaded() {
  window.dispatchEvent(new Event('resize'));
}
```

**Comment by sneas** [comment]
Hi @villyaraujo. The issue should be fixed in v2.3.1. Feel free to reopen the issue otherwise.
